scripts/serial_link.py

```diff
@@ -426,8 +453,12 @@ def format_log_json_entry(t0, item):
       link.add_global_callback(default_log_json_callback(log_file))
     else:
       link.add_global_callback(default_log_callback(log_file))
-    if args.reset:
-      link.send_message(ids.RESET, '')
+  if args.reset:
+    link.send_message(ids.RESET, '')
+  # Setup watchdog
+  watchdog = args.watchdog[0]
+  if watchdog:
+    link.add_callback(0xFFFF, Watchdog(float(watchdog), default_watchdog))
```

Comment:
Ha, yeah had tried that, didn't work. Looked closed, and have to do this to bring it in:

```python
from sbp_messages import SBP_HEARTBEAT
```

Isn't the wildcard in `sbp_piksi.py` doing that? Python, how does it work

Done.